platform/client-admin/src/dataFacade.tsx

```diff
@@ -406,8 +402,16 @@ export type Volunteer = Address & {
   city?: Maybe<Scalars['String']>;
   postcode?: Maybe<Scalars['Int']>;
   dateOfBirth?: Maybe<Scalars['DateTime']>;
-  canPhoneCall: Scalars['Boolean'];
-  canDeliver: Scalars['Boolean'];
+  /**
+   * This does not work for MongoDB - https://github.com/aerogear/graphback/issues/1241
+   * @db.default: false
```

Comment:
This is a minor issue but I have seen these annotations being propagated from the model till here, I am sure they are not of use and pollutes even further this file. Is this something we should consider here https://github.com/aerogear/graphback/issues/1085?

Reply:
Would like to know what you think? @wtrocki

Reply:
That will be gone. Nothing to worry about! Feel free to get that to master